lib/MonadicRewrite.thy

```diff
@@ -507,6 +507,30 @@ next
     by (fastforce simp: corres_underlying_def split_def)
 qed
 
+lemma monadic_rewrite_corres':
```

Comment:
This is another place where we're a bit inconsistent in naming schemes. Do people have a preference between `monadic_rewrite_corres` / `monadic_rewrite_corres'` and `monadic_rewrite_corres_lhs` / `monadic_rewrite_corres_rhs`?

Reply:
I guess there’s also the `xxx_l` and `xxx_r` version, too. I think using `’` to mean things about the concrete side is concise and nice, particularly when the thing itself is short, but it can be used to represent something else, too.

I am not sure how much effort we want to put in this particular case, though, since these rules are not used very much.